docs/source/tutorials/mapping_reads_reference_assembly.md

```diff
@@ -0,0 +1,190 @@
+
+Reads can be mapped as paired or single. If paired is used, then the matching regions are defined by the insert size and the length of each read
+
+![https://commons.wikimedia.org/wiki/File:Mapping_Reads.png](https://upload.wikimedia.org/wikipedia/commons/2/2e/Mapping_Reads.png)
+</br>
+</br>
+
+## 2. Why mapping reads
+
+Read mapping techniques are used for several purposes.
+Some examples:
+
+1) To evaluate the quality of an assembly (or to compare different methods used to assemble your reads). Read mapping can help identifying problematic areas in your assembly process. Indeed, statistics are necessary but might not be sufficient to evaluate the quality of your assembly.
+
+   To assess the quality of an assembly we can eg. to look at:
+   - the coverage regularity (ex: some repeated regions might have increased coverage)
+   - the coverage at the beginning and end of scaffolds - is it good enough?
+   - are there positions where pileup of reads show polymorphism?
+   - ...
+
+2) Assembly polishing software such as `pilon` and `reapr` use mapped reads to identify potential positions where the assemblies should be improved. It is good to visualize what information they are using.
+
+Assembly polishing softwares can eg. use variation in coverage, wrong read pairs orientation, discrepancy between expected insert size and actual insert size obtained from read mapping (ie. longer than expected) to improve assembly.
+
+3) To identify SNPs: some methods use reads mapping against a reference genome to identify and type variants/SNPs (e.g. [snippy](https://github.com/tseemann/snippy))
+</br>
+
+## 3. Practical: mapping reads to the assembly obtained from the same reads.
+
+We have assembled reads using `Bifrost pipeline`. The data for today's practical are found on SAGA in `/cluster/projects/nn9305k/tutorial/20210412_mapping_visualization/mapping_practical`
+
+Description of the files in this folder:
+- the raw_reads: `MiSeq_Ecoli_MG1655_50x_R{1,2}.fastq.gz`
+- the trimmed-reads: `MiSeq_Ecoli_MG1655_50x_{R1,R2,S}_concat_stripped_trimmed.fq.gz`
+- the assembly (polished with Pilon, output during prokka annoation): `MiSeq_Ecoli_MG1655_50x.fna` (fasta format)
+
+Mapping can be done either with raw reads or trimmed reads. If you are interested to see if there are positions with SNPs, it is easier to use trimmed-reads (avoiding the visualization noise due to the presence of adapters)
+
+To map reads, we use the [bwa mem] option from [bwa tools] software.
+
+NB: Convention: means `<change_me>`
+
+</br>
+<div style="background-color:silver">
+_**PRACTICAL EXERCISE:**_
+</br>
+In your `home` directory make a directory for today's work
+and a folder called `mapping` where you will **copy** the input files
+
+```bash
+cd <home>
+mkdir mapping
+cd mapping
+
+rsync -rauPW /cluster/projects/nn9305k/tutorial/20210412_mapping_visualization/mapping_practical/* .
```

Comment:
refer to a page with rsync

Reply:
This page needs to be made.  Removed this part